docs/model/type/user.md

```diff
@@ -0,0 +1,8 @@
+# Users
```

Comment:
Title and path of this document don't match how it is referred to earlier and in the text. i.e. It seems like this should either be consistently "user(s)" or "people", but not both interchangeably...

Reply:
Good catch - I actually changed from "People" to "Users" at the last minute, but didn't do a thorough job updating it everywhere. I'll fix this.